packages/acs-ui-common/review/acs-ui-common.api.md

```diff
@@ -28,19 +28,19 @@ export type CommonProperties<A, B> = {
     [P in keyof A & keyof B]: A[P] extends B[P] ? P : never;
 }[keyof A & keyof B];
 
-// @public (undocumented)
+// @public
 export const fromFlatCommunicationIdentifier: (id: string) => CommunicationIdentifier;
 
-// @public (undocumented)
+// @public
 export type FunctionWithKey<KeyT, ArgsT extends unknown[], RetT> = (key: KeyT, ...args: ArgsT) => RetT;
 
-// @public
-export const getApplicationId: () => string;
+// @internal
```

Comment:
Does @internal mean used in other packages in the repo but shouldn't be exported publically?

Reply:
Yes, for us.

According to api-extractor docs, this marks *public API* that is intended not be used except for internal purposes... https://api-extractor.com/pages/tsdoc/doc_comment_syntax/#release-tags

In our case, I think we should simply make sure that aren't any @internal in the communication-react.api.md (it's OK to have them in the packlet APIs)